extensions/objects/templates/skill-selector-editor.directive.ts

```diff
@@ -17,7 +17,7 @@
  */
 
 require(
-  'components/skill-selector/skill-selector.directive.ts');
+  'components/skill-selector/skill-selector.component.ts');
```

Comment:
not required.

Reply:
Going to wait as I'm curious about [this question](https://github.com/oppia/oppia/pull/11791#discussion_r587011631) before making all the changes.

Reply:
Leaving this `require` for now (from[ this comment)](https://github.com/oppia/oppia/pull/11791#discussion_r587057593)